x-pack/plugins/spaces/public/plugin.tsx

```diff
@@ -88,15 +96,15 @@ export class SpacesPlugin implements Plugin<SpacesPluginSetup, SpacesPluginStart
       spacesManager: this.spacesManager,
     });
 
-    plugins.spacesOss.registerSpacesApi(this.createSpacesApi());
+    plugins.spacesOss.registerSpacesApi(this.spacesApi);
 
-    return {};
+    return this.spacesApi;
```

Comment:
Thanks for calling this out--

I had to double check, but no consumers are using the spaces API from the setup contract. I think I added it while I was working on a commit because I thought I'd need it, and it turns out I didn't, and I simply forgot to remove it. I'll remove it now 😄